.circleci/config.yml

```diff
@@ -290,6 +290,17 @@ jobs:
       - *persist_to_workspace_step
```

Comment:
Side note: That 'up-to-date with master' check feels slightly superfluous. Github will warn and block PRs with conflicts. So having this check means people with good intentions (PRs) have to keep watching and updating their branches (via merge or rebase) when there really isn't a problem. Can the DD team consider disabling that? Given how rare reviews are (this would be the 2nd or 3rd time I've waited ~1 month for a review), it's yet another bump in the road.

Reply:
Thank you for the thoughtful feedback. The team here wants to smooth out more of the bumps that make community contributions presently difficult. We'll have a chance to discuss this soon and I'll be sure to add your feedback to inform how we improve our process.